src/features/side_by_side_wrap.rs

```diff
@@ -293,10 +294,23 @@ pub fn wrap_plusminus_block<'c: 'a, 'a>(
                 .unwrap_or_else(|| panic!("bad syntax alignment {}", errhint)),
             must_wrap,
             line_width,
-            &SyntectStyle::default(),
-            &config.inline_hint_color,
+            &config.null_syntect_style,
+            &Some(config.wrap_config.inline_hint_syntect_style),
         );
 
+        // TODO: Why is the background color set to white when
+        // ansi_term_style.background is None?
+        let inline_hint_style = if config
+            .inline_hint_style
+            .ansi_term_style
+            .background
+            .is_some()
+        {
+            Some(config.inline_hint_style)
+        } else {
+            None
+        };
```

Comment:
I was expecting that, if `style.background = None`, then no change to the background color should be made. But in fact what I'm seeing is that the  background color is set to white for the symbol character. I haven't understood that yet. My initial thought was not to use the `syntax_style_sections` at all, and instead only use the `diff_style_sections` (and in fact I have a [commit](https://github.com/dandavison/delta/commit/c5cde12e48a330c9b04a029e70987ce7f0185367) implementing that), but that wasn't possible due to this issue of the background color. In any case, this branch works and has an appropriate CLI, even if the implementation is a bit convoluted.

Reply:
I think you're probably aware of this issue already, seeing as I'm following your code in using both `syntax_style_sections` and `diff_style_sections`!

Reply:
Indeed, the interaction between these two colorings is quite something, good to know you also ran into that :)